packages/slate-react/src/components/editable.tsx

```diff
@@ -234,7 +234,47 @@ export const Editable = (
           return
         }
 
-        event.preventDefault()
+        let native = false
+        if (
+          type === 'insertText' &&
+          selection &&
+          Range.isCollapsed(selection)
+        ) {
+          // Only do it for single character events, for the simplest scenario,
+          // for now.
+          if (event.data && event.data.length === 1) {
+            native = true
+          }
+
+          // Chrome seems to have issues correctly editing the start of nodes.
+          // I see this when there is an inline element, like a link, and you select
+          // right after it (the start of the next node).
+          const { anchor } = selection
+          if (anchor.offset === 0) {
```

Comment:
Can you collapse this `anchor` if statement and the previous `event.data` if statement into the one above it, just to do it all in one go. (While preserving this great comment about Chrome compatibility.)